proto/raft_serverpb.proto

```diff
@@ -191,6 +191,7 @@ message MergedRecord {
     uint64 source_region_id = 1;
     metapb.RegionEpoch source_epoch = 2;
     repeated metapb.Peer source_peers = 3;
+    repeated metapb.Peer source_removed_records = 9;
```

Comment:
It's better to move it to the bottom.  Otherwise next change may still use 9.

Reply:
Why removed_records cannot be used for this purpose

Reply:
> Why removed_records cannot be used for this purpose

Because peers in `removed_records` belong target region, `source_removed_records` belong to source region.